# HG changeset patch
# User Rik <rik@octave.org>
# Date 1363876979 25200
#      Thu Mar 21 07:42:59 2013 -0700
# Node ID b8a4f313e78e016f6ba6966546567042992683e8
# Parent  d8babb249f65699f11328ccbb992736053d85c2d
copper.m: Make colormap compatible with Matlab (bug #36473).

* scripts/image/copper.m: Make colormap compatible with Matlab (bug #36473).

diff --git a/scripts/image/copper.m b/scripts/image/copper.m
--- a/scripts/image/copper.m
+++ b/scripts/image/copper.m
@@ -44,18 +44,18 @@ function map = copper (n)
   endif
 
   if (n == 1)
     map = [0, 0, 0];
   elseif (n > 1)
     x = linspace (0, 1, n)';
     r = (x < 4/5) .* (5/4 * x) ...
       + (x >= 4/5);
-    g = 4/5 * x;
-    b = 1/2 * x;
+    g = 0.7812 * x;
+    b = 0.4975 * x;
     map = [r, g, b];
   else
     map = zeros (0, 3);
   endif
 
 endfunction
 
 
